### ISSUE TITLE
JAWS not identifying Image role and Accessible label for “SVG Images” in both arrow key navigation and in quick navigation “g” in FF. 

### ISSUE THREAD

**Issue by Rasubra8** [FreedomScientific/standards-support#373]
### Summary

> Brief description of the issue. Please list any specific steps.
> 
JAWS not identifying Image role and the Accessible name for SVG image in both arrow key and quick Key navigation. But, identifying the same when listed the graphics in the page using "Insert+control+g". 

### Expected result
> What did you expect to happen?
Should announce the role graphic and the Accessible name provided using aria-label or alt attribute with image role. 

### Actual result
> What actually did happen?
Not identifying the graphic in both arrow key navigation and and in Quick navigation. Hence the user would not be aware of its presence. 

### Example

> A test case that demonstrates the issue - provide linked test case, CodePen, JSbin, etc...

This issue can be tested in the following link. 
https://weboverhauls.github.io/demos/svg/
#### JAWS version and build number
jaws 2019 and 2020

 
#### Operating System and version
windows 10

#### Browser and version:
FF73

**Comment by JAWS-test** [comment]
related: https://github.com/FreedomScientific/VFO-standards-support/issues/304

**Comment by stevefaulkner** [comment]
Closing this, please feel free to re-open if this still occurs with Latest (JAWS 2024).
